src/App.css

```diff
@@ -29,53 +127,40 @@ body {
   list-style: none;
 }
 
-.bookContainer {
-  margin: 1rem auto;
-  border: 1px solid black;
-  padding: 2rem;
-  width: 80%;
-}
-
-.bookContainer span {
-  margin: 0.8rem;
-}
-
-.delete {
-  float: right;
-}
-
 .formContainer {
-  margin-left: 8%;
-  height: 100vh;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+  margin: 6.25rem;
 }
 
 .addBook {
   color: gray;
   font-weight: bolder;
 }
 
+.form {
+  display: flex;
+  justify-content: space-between;
+}
+
 .formTitle {
-  padding-right: 22%;
+  padding-right: 15rem;
   padding-top: 0.5rem;
   padding-bottom: 0.5rem;
-  margin-right: 0.5rem;
+  margin-right: 0.1rem;
   border-radius: 2px;
   border: 2px solid silver;
 }
 
 .formAuthor {
   padding-top: 0.5rem;
+  padding-right: 7rem;
   padding-bottom: 0.5rem;
-  margin-right: 0.5rem;
+  margin-right: 0.1rem;
```

Comment:
- [ ] Following CSS best Practices, it would be better to use shorthand properties. Instead of using padding-top, padding-right, padding-bottom, and padding-left separately you can simply use just one line: `padding: 0.5 7rem 0.5rem 0;`